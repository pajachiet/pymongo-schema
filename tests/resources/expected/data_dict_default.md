
### Database: test_db1
#### Collection: test_col1 
|Field_compact_name     |Field_name        |Full_name     |Description     |Count     |Percentage     |Types_count                                   |
|-----------------------|------------------|--------------|----------------|----------|---------------|----------------------------------------------|
|_id                    |_id               |None          |None            |25359     |100.0          |oid : 25359                                   |
|address                |address           |None          |None            |25359     |100.0          |OBJECT : 25359                                |
| . building            |building          |None          |None            |25359     |100.0          |string : 25359                                |
| . coord               |coord             |None          |None            |25359     |100.0          |ARRAY(float : 50714, null : 2) : 25359        |
| . street              |street            |None          |None            |25359     |100.0          |string : 25359                                |
| . zipcode             |zipcode           |None          |None            |25359     |100.0          |string : 25359                                |
|borough                |borough           |None          |None            |25359     |100.0          |string : 25359                                |
|cuisine                |cuisine           |None          |None            |25359     |100.0          |string : 25359                                |
|grades                 |grades            |None          |None            |25359     |100.0          |ARRAY(OBJECT : 93463, null : 738) : 25359     |
| : date                |date              |None          |None            |93463     |368.56         |date : 93463                                  |
| : grade               |grade             |None          |None            |93463     |368.56         |string : 93463                                |
| : score               |score             |None          |None            |93463     |368.56         |integer : 93450, null : 13                    |
|name                   |name              |None          |None            |25359     |100.0          |string : 25359                                |
|restaurant_id          |restaurant_id     |None          |None            |25359     |100.0          |string : 25359                                |

#### Collection: test_col2 
|Field_compact_name     |Field_name        |Full_name     |Description     |Count     |Percentage     |Types_count                                   |
|-----------------------|------------------|--------------|----------------|----------|---------------|----------------------------------------------|
|_id                    |_id               |None          |None            |25359     |100.0          |oid : 25359                                   |
|address                |address           |None          |None            |25359     |100.0          |OBJECT : 25359                                |
| . building            |building          |None          |None            |25359     |100.0          |string : 25359                                |
| . coord               |coord             |None          |None            |25359     |100.0          |ARRAY(float : 50714, null : 2) : 25359        |
| . street              |street            |None          |None            |25359     |100.0          |string : 25359                                |
| . zipcode             |zipcode           |None          |None            |25359     |100.0          |string : 25359                                |
|borough                |borough           |None          |None            |25359     |100.0          |string : 25359                                |
|cuisine                |cuisine           |None          |None            |25359     |100.0          |string : 25359                                |
|grades                 |grades            |None          |None            |25359     |100.0          |ARRAY(OBJECT : 93463, null : 738) : 25359     |
| : date                |date              |None          |None            |93463     |368.56         |date : 93463                                  |
| : grade               |grade             |None          |None            |93463     |368.56         |string : 93463                                |
| : score               |score             |None          |None            |93463     |368.56         |integer : 93450, null : 13                    |
|name                   |name              |None          |None            |25359     |100.0          |string : 25359                                |
|restaurant_id          |restaurant_id     |None          |None            |25359     |100.0          |string : 25359                                |


### Database: test_db2
#### Collection: test_col 
|Field_compact_name     |Field_name        |Full_name     |Description     |Count     |Percentage     |Types_count                                   |
|-----------------------|------------------|--------------|----------------|----------|---------------|----------------------------------------------|
|_id                    |_id               |None          |None            |25359     |100.0          |oid : 25359                                   |
|address                |address           |None          |None            |25359     |100.0          |OBJECT : 25359                                |
| . building            |building          |None          |None            |25359     |100.0          |string : 25359                                |
| . coord               |coord             |None          |None            |25359     |100.0          |ARRAY(float : 50714, null : 2) : 25359        |
| . street              |street            |None          |None            |25359     |100.0          |string : 25359                                |
| . zipcode             |zipcode           |None          |None            |25359     |100.0          |string : 25359                                |
|borough                |borough           |None          |None            |25359     |100.0          |string : 25359                                |
|cuisine                |cuisine           |None          |None            |25359     |100.0          |string : 25359                                |
|grades                 |grades            |None          |None            |25359     |100.0          |ARRAY(OBJECT : 93463, null : 738) : 25359     |
| : date                |date              |None          |None            |93463     |368.56         |date : 93463                                  |
| : grade               |grade             |None          |None            |93463     |368.56         |string : 93463                                |
| : score               |score             |None          |None            |93463     |368.56         |integer : 93450, null : 13                    |
|name                   |name              |None          |None            |25359     |100.0          |string : 25359                                |
|restaurant_id          |restaurant_id     |None          |None            |25359     |100.0          |string : 25359                                |

