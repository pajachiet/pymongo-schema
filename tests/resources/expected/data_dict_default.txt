
### Database: test_db1
--- Collection: test_col1 
Field_compact_name Field_name        Count    Percentage Types_count                                  
_id                _id              25359    100.0       oid : 25359                                 
address            address          25359    100.0       OBJECT : 25359                              
 . building        building         25359    100.0       string : 25359                              
 . coord           coord            25359    100.0       ARRAY(float : 50714, null : 2) : 25359      
 . street          street           25359    100.0       string : 25359                              
 . zipcode         zipcode          25359    100.0       string : 25359                              
borough            borough          25359    100.0       string : 25359                              
cuisine            cuisine          25359    100.0       string : 25359                              
grades             grades           25359    100.0       ARRAY(OBJECT : 93463, null : 738) : 25359   
 : date            date             93463    368.56      date : 93463                                
 : grade           grade            93463    368.56      string : 93463                              
 : score           score            93463    368.56      integer : 93450, null : 13                  
name               name             25359    100.0       string : 25359                              
restaurant_id      restaurant_id    25359    100.0       string : 25359

--- Collection: test_col2 
Field_compact_name Field_name        Count    Percentage Types_count                                  
_id                _id              25359    100.0       oid : 25359                                 
address            address          25359    100.0       OBJECT : 25359                              
 . building        building         25359    100.0       string : 25359                              
 . coord           coord            25359    100.0       ARRAY(float : 50714, null : 2) : 25359      
 . street          street           25359    100.0       string : 25359                              
 . zipcode         zipcode          25359    100.0       string : 25359                              
borough            borough          25359    100.0       string : 25359                              
cuisine            cuisine          25359    100.0       string : 25359                              
grades             grades           25359    100.0       ARRAY(OBJECT : 93463, null : 738) : 25359   
 : date            date             93463    368.56      date : 93463                                
 : grade           grade            93463    368.56      string : 93463                              
 : score           score            93463    368.56      integer : 93450, null : 13                  
name               name             25359    100.0       string : 25359                              
restaurant_id      restaurant_id    25359    100.0       string : 25359


### Database: test_db2
--- Collection: test_col 
Field_compact_name Field_name        Count    Percentage Types_count                                  
_id                _id              25359    100.0       oid : 25359                                 
address            address          25359    100.0       OBJECT : 25359                              
 . building        building         25359    100.0       string : 25359                              
 . coord           coord            25359    100.0       ARRAY(float : 50714, null : 2) : 25359      
 . street          street           25359    100.0       string : 25359                              
 . zipcode         zipcode          25359    100.0       string : 25359                              
borough            borough          25359    100.0       string : 25359                              
cuisine            cuisine          25359    100.0       string : 25359                              
grades             grades           25359    100.0       ARRAY(OBJECT : 93463, null : 738) : 25359   
 : date            date             93463    368.56      date : 93463                                
 : grade           grade            93463    368.56      string : 93463                              
 : score           score            93463    368.56      integer : 93450, null : 13                  
name               name             25359    100.0       string : 25359                              
restaurant_id      restaurant_id    25359    100.0       string : 25359

